### PR TITLE
fix(misconf): Allow quotes in Dockerfile WORKDIR when detecting relative dirs

### LINKED ISSUE #2622: Using double quotes on WORKDIR in Dockerfile results in HIGH failure
## Description
Using the config scan on Dockerfiles results in a HIGH failure if the WORKDIR is wrapped in double quotes. If I remove the double quotes around it, everything is fine. 

## What did you expect to happen?
That trivy does not raise a HIGH failure if I wrap the WORKDIR in double quotes. 

## What happened instead?
It raised HIGH failure. 

## Output of run with `-debug`:
```
2022-07-28T19:30:35.955+0200	DEBUG	Severities: UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL
2022-07-28T19:30:35.996+0200	DEBUG	cache dir:  /Users/.../Library/Caches/trivy
2022-07-28T19:30:35.996+0200	INFO	Misconfiguration scanning is enabled
2022-07-28T19:30:36.504+0200	DEBUG	OS is not detected.
2022-07-28T19:30:36.504+0200	INFO	Detected config files: 5
2022-07-28T19:30:36.504+0200	DEBUG	Scanned config file: Chart/templates/spc.yaml
2022-07-28T19:30:36.504+0200	DEBUG	Scanned config file: Chart/templates/config.yaml
2022-07-28T19:30:36.504+0200	DEBUG	Scanned config file: Chart/templates/deployment.yaml
2022-07-28T19:30:36.504+0200	DEBUG	Scanned config file: Chart/templates/service.yaml
2022-07-28T19:30:36.504+0200	DEBUG	Scanned config file: Dockerfile

Dockerfile (dockerfile)

Tests: 22 (SUCCESSES: 21, FAILURES: 1, EXCEPTIONS: 0)
Failures: 1 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 1, CRITICAL: 0)

HIGH: WORKDIR path '"/src/omitted.WebApi"' should be absolute
══════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════
For clarity and reliability, you should always use absolute paths for your WORKDIR.

See https://avd.aquasec.com/misconfig/ds009
──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
 Dockerfile:27
──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
  27 [ WORKDIR "/src/ommited.WebApi"
──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────

## Output of `trivy -v`:

```
Version: 0.29.2
Vulnerability DB:
  Version: 2
  UpdatedAt: 2022-07-28 12:08:50.011144915 +0000 UTC
  NextUpdate: 2022-07-28 18:08:50.011144515 +0000 UTC
  DownloadedAt: 2022-07-28 16:36:36.630277 +0000 UTC
```

## Additional details (base image name, container registry info...):
Base `image: mcr.microsoft.com/dotnet/aspnet:6.0-alpine` which is probably not relevant anyways

### PR DESCRIPTION
**Pull request by liamg**
Resolves #2622

Signed-off-by: Liam Galvin <liam.galvin@aquasec.com>

## Description
Regular expression for WORKDIR used in the rego rule in defsec did not allow quotes - now it does.

## Related issues
- Close #2622

### CODE REVIEW

File: integration/testdata/helm.json.golden [hunk 1]
```diff
@@ -268,7 +268,7 @@
       "Class": "config",
       "Type": "helm",
       "MisconfSummary": {
-        "Successes": 34,
+        "Successes": 78,
```
Comment (knqyf263) [inline integration/testdata/helm.json.golden:271]: Did you add a lot of checks for Kubernetes?

Comment (liamg) [reply integration/testdata/helm.json.golden:271]: Haha, yes! @josedonizetti added lots of CIS Benchmark checks in https://github.com/aquasecurity/defsec/pull/720 and https://github.com/aquasecurity/defsec/pull/721 :)

Comment (knqyf263) [reply integration/testdata/helm.json.golden:271]: Oh, cool👍 I just wanted to confirm that a number of `Successes` didn't jump by mistake. Thanks for confirming.

**Review by knqyf263 — APPROVED** [review]

**Comment by knqyf263** [comment]
I'll wait for @owenrumney's review.

**Review by owenrumney — APPROVED** [review]

**Pull request merged**